en/guide/extending_csd.md

```diff
@@ -1,6 +1,121 @@
 # Extending Camera Streaming Daemon
 
-## Create a Custom Video Stream
+## Create a Custom Camera Device
+
+Camera-Streaming-Daemon has native support for auto detection of video cameras exported as Video4Linux(V4L2) devices. This covers most of regular Linux cameras, but there are some cameras that don't have V4L2 support or cases where user wants to set very specific parameters or do custom post-processing in video before exporting.
```

Comment:
"exported as Video4Linux(V4L2) devices" or "that support the Video4Linux (V4L2) API"? Either way, space before "(".

What would be a parameter that someone might want to support?

Reply:
3. Change the text to "that support the Video4Linux (V4L2) API" with space before "("
> What would be a parameter that someone might want to support?

Custom IOCTL declared by ISP/driver. Most of the high-end camera drivers have such custom IOCTLs to control/tune the ISP/sensor

Reply:
15. Explain in more detail what type of V4L2 device is supported and what is not